x-pack/libbeat/tests/system/test_management.py

```diff
@@ -152,8 +153,9 @@ def test_fetch_configs(self):
 
         proc.check_kill_and_wait()
 
-    @unittest.skipIf(not INTEGRATION_TESTS,
-                     "integration tests are disabled, run with INTEGRATION_TESTS=1 to enable them.")
+    # @unittest.skipIf(not INTEGRATION_TESTS,
+    #                  "integration tests are disabled, run with INTEGRATION_TESTS=1 to enable them.")
```

Comment:
Do these 2 lines need to be commented out?

Reply:
they will be back when the skip is removed